src/library/scala/collection/immutable/HashMap.scala

```diff
@@ -317,6 +328,20 @@ object HashMap extends ImmutableMapFactory[HashMap] with BitOperations.Int {
         None
     }
 
+    override protected def contains0(key: A, hash: Int, level: Int): Boolean = {
+      // Note: this code is duplicated with `get0`
+      val index = (hash >>> level) & 0x1f
+      val mask = (1 << index)
+      if (bitmap == - 1) {
+        elems(index & 0x1f).contains0(key, hash, level + 5)
```

Comment:
In this code, it was certainly masked and the extra masking does nothing, and the JIT compiler will almost surely elide it, but why make it do the extra work?  Good catch.